JDTeamAcetabulum/qna
# Students can see all the questions and answers they have answered

## Changes 
This allows students to be able to see all the questions and answer they have answered. it also allows them to see whether it was correct or not.

## Context 
- the teacher must already have asked a question and student must have answered at least one question.

## Testing 
- login as a student
- answer the question asked
- got to statistics tab then to view answered questions
- check whether it is visible or not

## Pre-merge checklist 

- [ ] Build succeeds locally and in CI
- [ ] Proper documentation
- [ ] Git history clean
